### PR TITLE
Update MODEL_FIELD_ATTRIBUTE_NAME to allow for native types.

### LINKED ISSUE #6: Support for Native types mapping
Using:
Webstorm 2021.1
Prisma support 1.0.3.201
Prisma 2.21.2

Native types mapping like @db.Text doesn't work with this plugin.
description String? @db.Text
throws this message:

<model block statement>, PrismaTokenType.DOUBLE_COMMENT, PrismaTokenType.ENTITY_NAME, PrismaTokenType.EOL, PrismaTokenType.TRIPLE_COMMENT or PrismaTokenType.} expected, got 'description'

### PR DESCRIPTION
**Pull request by JarekToro**
`@db.Timestamp(6)`

Fixes #6

### CODE REVIEW

File: src/main/grammars/PrismaLexer.flex [hunk 1]
```diff
@@ -68,4 +68,4 @@ FUNCTION_NAME=[a-zA-Z]+
 
 {EOL}         { return EOL; }
 {WHITE_SPACE} { return WHITE_SPACE; }
-[^] { return BAD_CHARACTER; }
\ No newline at end of file
+[^] { return BAD_CHARACTER; }
```
Comment (JarekToro) [inline src/main/grammars/PrismaLexer.flex:71]: Honestly I'm not sure why this line is showing its been edited. I even went into my IDE to see if it was a line ending thing. But my IDE doesn't show the same line change so ¯\_(ツ)_/¯

The only line that I found needed to change was 34

Comment (hiddenist) [reply src/main/grammars/PrismaLexer.flex:71]: The difference is that your file has a trailing line break, and the original file does not. Your editor is probably configured to append it.

**Comment by hiddenist** [comment]
I made a fork and was going to add this same thing for the same reason. I would love to see this merged.

**Comment by olance** [comment]
OMG so sorry about the delay 🙈 

Thanks a lot @JarekToro for the PR, let's merge this and I'll try to release a new version this week-end.

**Pull request merged**